server/vulnerabilities/nvd/tools/README.md

```diff
@@ -126,7 +126,7 @@ host2.foo.bar CVE-2017-8817 cpe:/a:haxx:curl:7.55.0
 
 ### `csv2cpe`
 
-*csv2cpe* is a tool that generates an URI-bound CPE from CSV input, flags configure the meaning of each input field:
+*csv2cpe* is a tool that generates a URI-bound CPE from CSV input, flags configure the meaning of each input field:
```

Comment:
If it starts with a "yoo" sound, use "a" (e.g. a user story, a US company)
If it starts with a "uh" sound, use "an" (e.g. an upcoming sprint, an unauthorized vendor)